gui/services/migrations/0023_configurable_netdata_metrics.py

```diff
@@ -0,0 +1,33 @@
+from django.db import migrations, models
+import freenasUI.freeadmin.models.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('services', '0022_remove_smb_doscharset'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='NetDataGlobalSettings',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('history', models.IntegerField(default=86400)),
+                ('memory_mode', models.CharField(default='save', max_length=10)),
+                ('update_every', models.IntegerField(default=1)),
+                ('http_port_listen_backlog', models.IntegerField(default=100)),
+                ('bind_to', freenasUI.freeadmin.models.fields.ListField(default=['*'])),
```

Comment:
this should probably be named `bindip` or jsut `bind` to conform with other services.